library/std/src/sync/once.rs

```diff
@@ -123,6 +124,9 @@ unsafe impl Sync for Once {}
 #[stable(feature = "rust1", since = "1.0.0")]
 unsafe impl Send for Once {}
 
+#[stable(feature = "ref_unwind_safe", since = "1.56.1")]
```

Comment:
(Version will need to be updated.)

Reply:
Also, should this use a new feature—like `sync_once_ref_unwind_safe`—to avoid confusion?